internal/utils/file/file.go

```diff
@@ -271,7 +271,7 @@ func getPathsByExtension(projectPath, subPath string, extensions ...string) ([]s
 // The line and the dependency trimmed is also returned.
 //
 //nolint:funlen,gocyclo
-func getDependencyInfo(paths []string, dependency string) (string, string, string) {
+func GetDependencyInfo(paths []string, dependency string) (string, string, string) {
```

Comment:
Also update godoc  from `getDependencyInfo` to `GetDependencyInfo`

Reply:
I was thinking about this function signature, and would be better for readability to switch parameters orders and make `path []string` variadic.

```go
func GetDependencyInfo(dependency string, paths ...string) (string, string, string)
....
}
```
So, the caller [here](https://github.com/ZupIT/horusec/blob/2627656ab166fc1e5f65befcafd9758f7287ce70/internal/utils/file/file.go#L244) can be changed to `return GetDependencyInfo(dependency, paths...)`

And [here](https://github.com/ZupIT/horusec/blob/2627656ab166fc1e5f65befcafd9758f7287ce70/internal/services/formatters/generic/trivy/formatter.go#L169) we can simply call `file.GetDependencyInfo(addVuln.Code, target)`

Reply:
@matheusalcantarazup it looks more readable, I agree with that.